src/net/sourceforge/kolmafia/request/NPCPurchaseRequest.java

```diff
@@ -910,6 +910,11 @@ public static final void parseShopResponse(final String urlString, final String
       }
     }
 
+    if (shopId.equals("dino")) {
```

Comment:
Right. I was going to say we alphabetize shopids, but, no, we don't. :)

However, you also need to add this shopid in registerShopRequest farther down in this same file.

Reply:
okay, hope I got that right.

I checked "marioitems" above and that one wasn't further down. I'm not sure whether it should be, or how many are missing.